composer.json

```diff
@@ -4,11 +4,11 @@
 	"type": "wordpress-plugin",
 	"license": "GPL-2.0-only",
 	"require": {
-		"php": ">=5.6"
+		"php": ">=7.3"
```

Comment:
Officially, PHP does not support 7.3 anymore, should I even bump it to 7.4 ?
https://www.php.net/supported-versions.php